addons/web/__manifest__.py

```diff
@@ -282,7 +282,7 @@
             'web/static/src/legacy/js/report/report.js',
         ],
         'web.report_assets_pdf': [
-            'web/static/src/css/reset.min.css',
+            'web/static/src/legacy/js/report/reset.min.css',
```

Comment:
yep, we moved the file in 0573acae2306bf5da2005852da9323ddc59e5431, but forgot to update this line.